packages/experimental-app-router/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "peerDependencies": {
     "@apollo/client": ">=3.8.0",
-    "@apollo/experimental-nextjs-app-support": ">=0.5.0",
```

Comment:
I'm not sure if we should update the peer dependency's version here, that would be a breaking change.

https://github.com/semver/semver/issues/502#issuecomment-466501843

By upgrading the example project and the dev dependency we are still avoiding the dependabot vulnerability but giving users the choice to use 0.5.0 through 0.8.0.

Reply:
Its a v1.0 change so the risk is justified. Also we cannot allow the user to have a vulnerable version anyway

Reply:
Agreed. We should still increment `experimental-app-router` with a `major` bump though.